parlai/core/agents.py

```diff
@@ -416,18 +416,6 @@ def compare_init_model_opts(opt, curr_opt):
         print('*' * 75)
 
 
-def _load_opt_file(optfile):
```

Comment:
this function is called in https://github.com/facebookresearch/ParlAI/blob/cdfa62dfb2eae7bd23bcba02ab42a45f7f5118d6/parlai/agents/bert_classifier/bert_classifier.py#L12

so you'll have to fix the function name & correct import there

Reply:
done